prerequisites/install-functions/report_results.sh

```diff
@@ -15,7 +15,7 @@ report_results()
 
   type_CMAKE=`type ${CMAKE}`
   fully_qualified_CMAKE="/${type_CMAKE#*/}"
```

Comment:
Lines like the two above should probably be replaced with the simpler and easier to maintain: `fully_qualified_CMAKE="$(type -P cmake)"` (or replace `cmake` with `${CMAKE}`, but it's not immediately obvious to me what the point of the indirection is here...